src/clj/rems/api/catalogue.clj

```diff
@@ -4,17 +4,20 @@
             [rems.api.util] ; required for route :roles
             [rems.db.catalogue :as catalogue]
             [ring.util.http-response :refer :all]
-            [schema.core :as s]))
+            [schema.core :as s]
+            [rems.config :refer [env]]
+            [rems.api.util/check-roles :as check-roles ]))
```

Comment:
this should be `[rems.api.util :refer [check-roles]]`

Reply:
ok!